app/app/urls.py

```diff
@@ -165,6 +165,11 @@
         name='request_verify_google'
     ),
     url(r'^api/v0.1/profile/verify_user_google', dashboard.views.verify_user_google, name='verify_user_google'),
+    url(
+        r'^api/v0.1/profile/verify_ens',
+        dashboard.views.verify_profile_with_ens,
+        name='verify_with_ens'
+    ),
```

Comment:
I feel while we're trying to keep legacy code from breaking, we might want to write new code bits in the modern way. E.g, using path instead of url.